packages/babel-plugin/src/styled/__tests__/index.test.tsx

```diff
@@ -100,27 +100,25 @@ describe('styled component transformer', () => {
     expect(actual).toInclude('"--var-test-propscolor": (props.color || "") + "px"');
   });
 
-  xit('should add react default import if missing', () => {
+  it('should add react default import if missing', () => {
     const actual = transform(`
       import { styled } from '@compiled/core';
       const ListItem = styled.div\`
         font-size: 20px;
       \`;
     `);
-
-    expect(actual).toInclude('import React from "react";');
+    expect(actual).toInclude(`${transform("import * as React from 'react'")}`);
```

Comment:
I found it a bit weird to add the `transform` here as well but otherwise the strings didn't match 🤔